lib/features/stories/view/screens/show_taken_image_screen.dart

```diff
@@ -61,35 +67,28 @@ class _ShowTakenStoryScreenState extends ConsumerState<ShowTakenImageScreen> {
     }
   }
 
+  Future<void> saveCroppedFile(
+      Uint8List croppedFile, Function(File) onFileSaved) async {
+    if (croppedFile.isNotEmpty) {
+      Directory tempDir = await getTemporaryDirectory();
+      String fileName = const Uuid().v4();
+      String filePath = '${tempDir.path}/$fileName.jpg';
+      final File file = await File(filePath).writeAsBytes(croppedFile);
+      onFileSaved(file);
+    }
+  }
+
   Future<void> _cropImage() async {
     try {
-      if (_imageFile == null || !await _imageFile!.exists()) {
-        ScaffoldMessenger.of(context).showSnackBar(
-          const SnackBar(content: Text('Image file not found.')),
-        );
-        return;
-      }
-
-      final croppedFile = await ImageCropper().cropImage(
-        sourcePath: widget.image.path,
-        uiSettings: [
-          AndroidUiSettings(
-            toolbarTitle: 'Cropper',
-            toolbarColor: Palette.secondary,
-            toolbarWidgetColor: Colors.white,
-            aspectRatioPresets: [
-              CropAspectRatioPreset.original,
-              CropAspectRatioPreset.square,
-              CropAspectRatioPresetCustom(),
-            ],
-          ),
-          WebUiSettings(context: context),
-        ],
+      final croppedFile = await Navigator.of(context).push(
```

Comment:
I don't know how it worked in the demo, but, as we are using GoRouter, you can't use Navigator.
change to GoRouter.

Reply:
ok resolved